### ISSUE TITLE
Config remark display - Android

### ISSUE THREAD

**Issue by bia-pain-bache** [hiddify/hiddify-next#1077]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

When numbers involved in remarks, hiddify couldn't render them properly according to attached image:

![Screenshot_20240718_151813](https://github.com/user-attachments/assets/64f080ed-b0c4-46d3-ad59-a73fe07e61b3)

### Minimal Reproducible Example (MRE)

1. pick a subscription or config which has numbers in remarks.
2. import and connect
3. check proxy section

### Version

v1.6.3

### Platform/OS

Android

### Are you willing to submit a PR? If you know how to fix the bug.

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by eXense-auth** [comment]
still relevant for me
too big character spacing and black numbers on dark background design are not readable

**Comment by bia-pain-bache** [comment]
Unfortunately apk files are missing.

**Comment by bia-pain-bache** [comment]
No, still the same, updated app, cleared cache an data and nothig changed.
